wire-ios/Wire-iOS/Sources/Components/Buttons/IconButton.swift

```diff
@@ -302,6 +304,20 @@ class IconButton: ButtonWithLargerHitArea {
         tintColor = iconColor(for: state)
     }
 
+    private func updateBackgroundImageColor() {
+        let color = backgroundColor(for: state)?.resolvedColor(with: traitCollection)
+        if color != nil {
+            setBackgroundImage(UIImage.singlePixelImage(with: color!), for: state)
+
+            if adjustBackgroundImageWhenHighlighted, state.contains(.normal) {
+                setBackgroundImage(
+                    UIImage.singlePixelImage(with: color!.mix(UIColor.black, amount: 0.4)),
+                    for: .highlighted
+                )
```

Comment:
suggestion: unwrap the value:

```suggestion
        if let color = backgroundColor(for: state)?.resolvedColor(with: traitCollection) {
            setBackgroundImage(UIImage.singlePixelImage(with: color), for: state)

            if adjustBackgroundImageWhenHighlighted, state.contains(.normal) {
                setBackgroundImage(
                    UIImage.singlePixelImage(with: color.mix(UIColor.black, amount: 0.4)),
                    for: .highlighted
                )
```

Reply:
Done, thanks for the suggestion!